packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -15,35 +15,47 @@ const trace = makeTracer('NewAuction', true);
 /**
  * @param {import('./econ-behaviors.js').EconomyBootstrapPowers &
  *   interlockPowers} powers
+ * @param {{ options: { auctionsRef: { bundleID: string } } }} options
  */
-export const addAuction = async ({
-  consume: {
-    zoe,
-    board,
-    chainTimerService,
-    priceAuthority,
-    chainStorage,
-    economicCommitteeCreatorFacet: electorateCreatorFacet,
-    auctioneerKit: legacyKitP,
-  },
-  produce: { newAuctioneerKit, auctionsUpgradeComplete },
-  instance: {
-    consume: { reserve: reserveInstance },
-  },
-  installation: {
+export const addAuction = async (
+  {
     consume: {
-      auctioneer: auctionInstallation,
-      contractGovernor: contractGovernorInstallation,
+      zoe,
+      board,
+      chainTimerService,
+      priceAuthority,
+      chainStorage,
+      economicCommitteeCreatorFacet: electorateCreatorFacet,
+      auctioneerKit: legacyKitP,
+    },
+    produce: { newAuctioneerKit, auctionsUpgradeComplete },
```

Comment:
in discussion, we agreed `newAuctioneerKit` should go away in favor of the `auctionsUpgradeComplete` sync point and replacing `auctioneerKit` ( with reset, resolve ).

Reply:
done